apps/project/serializers.py

```diff
@@ -147,18 +149,24 @@ def validate_project(self, project):
         return project
 
     def validate(self, data):
+        data['project_id'] = int(self.context['view'].kwargs['project_id'])
+        # get the members of the project
+        members = ProjectMembership.objects.filter(
+            project=data['project_id']
+        ).values('member')
+        member = data.get('member')
+        if member:
+            member = member.id
+        if member in [mem['member'] for mem in members]:
+            raise serializers.ValidationError({'member': 'Member already exist'})
```

Comment:
Separate this validation to a different function.

Further, use something like below,

```
if ProjectMembership.objects.filter(project=data['project_id']).filter(member=data.get('member')).exists():
    raise ...
```

Reply:
Done!